force-app/main/default/classes/Log.cls

```diff
@@ -716,5 +771,99 @@ public without sharing class Log {
             this.response = response;
         }
     }
-
-}
\ No newline at end of file
+    
+    public class PostProcessingControlsBuilder {
+
+		private final Map<String, Boolean> controls;
+
+		private final String STACK_TRACE_KEY = 'stackTrace';
+		private final String USER_INFO_KEY = 'userInfo';
+		private final String OBJECT_INFO_KEY = 'objectInfo';
+		private final String RELATED_OBJECT_TYPE_KEY = 'relatedObjectType';
+		private final String RELATED_OBJECTS_KEY = 'relatedObjects';
+		private final String FLOW_VERSION_ID_KEY = 'flowVersionId';
+		private final String DEPLOY_RESULT_KEY = 'deployResult';
+		private final String AUDIT_TRAIL_KEY = 'auditTrail';
+		private final String PENDING_JOBS_KEY = 'pendingJobs';
+		private final String TOTAL_PENDING_JOBS_KEY = 'totalPendingJobs';
+		private final String TOTAL_ACTIVE_SESSION_KEY = 'totalActiveSession';
+		private final String INSTALLED_PACKAGES_KEY = 'installedPackages';
+
+		private PostProcessingControlsBuilder() {
+			controls = new Map<String, Boolean>{
+					STACK_TRACE_KEY => false,
+					USER_INFO_KEY => false,
+					OBJECT_INFO_KEY => false,
+					RELATED_OBJECT_TYPE_KEY => false,
+					RELATED_OBJECTS_KEY => false,
+					FLOW_VERSION_ID_KEY => false,
+					DEPLOY_RESULT_KEY => false,
+					AUDIT_TRAIL_KEY => false,
+					PENDING_JOBS_KEY => false,
+					TOTAL_PENDING_JOBS_KEY => false,
+					TOTAL_ACTIVE_SESSION_KEY => false,
+					INSTALLED_PACKAGES_KEY => false
+			};
+		}
+		public PostProcessingControlsBuilder setAll(Boolean value) {
+			for (String key : controls.keySet()) {
+				controls.put(key, value);
+			}
+			return this;
+		}
+		public PostProcessingControlsBuilder stackTrace(Boolean value) {
+			controls.put(STACK_TRACE_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder userInfo(Boolean value) {
+			controls.put(USER_INFO_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder objectInfo(Boolean value) {
+			controls.put(OBJECT_INFO_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder relatedObjectType(Boolean value) {
+			controls.put(RELATED_OBJECT_TYPE_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder relatedObjects(Boolean value) {
+			controls.put(RELATED_OBJECTS_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder flowVersionId(Boolean value) {
+			controls.put(FLOW_VERSION_ID_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder deployResult(Boolean value) {
+			controls.put(DEPLOY_RESULT_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder auditTrail(Boolean value) {
+			controls.put(AUDIT_TRAIL_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder pendingJobs(Boolean value) {
+			controls.put(PENDING_JOBS_KEY, value);
+			return this;
+		}
+		public PostProcessingControlsBuilder totalPendingJobs(Boolean value) {
```

Comment:
@mikita-kniazeu do we need one for total jobs? if we're getting pending jobs, it should automatically populate the count